test/e2e/crs/mocked_agent.yaml

```diff
@@ -0,0 +1,15 @@
+apiVersion: runtime.agentic-layer.ai/v1alpha1
+kind: Agent
+metadata:
+  labels:
+    app.kubernetes.io/name: agent-runtime-operator
+    app.kubernetes.io/managed-by: kustomize
+  name: mocked-agent
+spec:
+  framework: google-adk
+  image: wiremock/wiremock:latest
```

Comment:
Please do not use latest tag, but versioned tag. For curl it may be ok to use latest, as no breaking changes are expected. But I would not trust wiremock to keep stable all the time.